### ISSUE TITLE
[PATCH 5.6.0 README.win32] very minor typos

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#1743]
Migrated from [rt.perl.org#2994](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=2994) (status was 'resolved')

Searchable as RT2994$

**Comment by p5pRT** [comment]
### From yardley@tanet.net

Please note\, most of these are very minor\.  Feel free to do with
then as you please\.

Also\, I have two questions regarding README\.win32&#8203;:  \(1\) Do we want
to change the references to "manpage" to "perldoc"? and \(2\) Do we
want to include\, where relevant\, references to Windows 2000\, along
side of the references to Windows NT?

If so\, please let me know\, and I'll make the corrections\.
Otherwise\, I'll just leave them as they are\.

/acy

<details><summary>Inline Patch</summary>

```diff
--- README.win32.old    Mon Apr 03 07:14:51 2000
+++ README.win32        Mon Apr 03 11:14:49 2000
@@ -14,7 +14,7 @@
 =head1 DESCRIPTION
 
 Before you start, you should glance through the README file
-found in the top-level directory where the Perl distribution
+found in the top-level directory to which the Perl distribution
 was extracted.  Make sure you read and understand the terms under
 which this software is being distributed.
 
@@ -28,10 +28,10 @@
 
 You may also want to look at two other options for building
 a perl that will work on Windows NT:  the README.cygwin and
-README.os2 files, which each give a different set of rules to build
-a Perl that will work on Win32 platforms.  Those two methods will
-probably enable you to build a more Unix-compatible perl, but you
-will also need to download and use various other build-time and
+README.os2 files, each of which give a different set of rules to
+build a Perl that will work on Win32 platforms.  Those two methods
+will probably enable you to build a more Unix-compatible perl, but
+you will also need to download and use various other build-time and
 run-time support software described in those files.
 
 This set of instructions is meant to describe a so-called "native"
@@ -48,9 +48,9 @@
 for it is still experimental.  (Older versions of GCC are known
 not to work.)
 
-This port currently supports MakeMaker (the set of modules that
-is used to build extensions to perl).  Therefore, you should be
-able to build and install most extensions found in the CPAN sites.
+This port currently supports MakeMaker (the set of modules is used
+to build extensions to perl).  Therefore, you should be able to
+build and install most extensions found in the CPAN sites.
 See L<Usage Hints> below for general hints about this.
 
 =head2 Setting Up
@@ -70,9 +70,9 @@
 
     http://www.cpan.org/authors/id/GSAR/dmake-4.1pl1-win32.zip
 
-(This is a fixed version of original dmake sources obtained from
+(This is a fixed version of the original dmake sources obtained from
 http://www.wticorp.com/dmake/.  As of version 4.1PL1, the original
-sources did not build as shipped, and had various other problems.
+sources did not build as shipped and had various other problems.
 A patch is included in the above fixed version.)
 
 Fetch and install dmake somewhere on your path (follow the instructions
@@ -97,20 +97,20 @@
 =item Borland C++
 
 If you are using the Borland compiler, you will need dmake.
-(The make that Borland supplies is seriously crippled, and will not
+(The make that Borland supplies is seriously crippled and will not
 work for MakeMaker builds.)
 
-See L/"Make"> above.
+See L</"Make"> above.
 
 =item Microsoft Visual C++
 
 The nmake that comes with Visual C++ will suffice for building.
-You will need to run the VCVARS32.BAT file usually found somewhere
+You will need to run the VCVARS32.BAT file, usually found somewhere
 like C:\MSDEV4.2\BIN.  This will set your build environment.
 
-You can also use dmake to build using Visual C++, provided:
+You can also use dmake to build using Visual C++; provided, however,
 you set OSRELEASE to "microsft" (or whatever the directory name
-under which the Visual C dmake configuration lives) in your environment,
+under which the Visual C dmake configuration lives) in your environment
 and edit win32/config.vc to change "make=nmake" into "make=dmake".  The
 latter step is only essential if you want to use dmake as your default
 make for building extensions using MakeMaker.
@@ -125,7 +125,7 @@
 
 Make sure you install the binaries that work with MSVCRT.DLL as indicated
 in the README for the GCC bundle.  You may need to set up a few environment
-variables (usually run from a batch file).
+variables (usually ran from a batch file).
 
 You also need dmake.  See L</"Make"> above on how to get it.
 
@@ -145,12 +145,12 @@
 
 =item *
 
-Edit the makefile.mk (or Makefile, if using nmake) and change the values
-of INST_DRV and INST_TOP.   You can also enable various build
-flags.  These are explained in the makefiles.
+Edit the makefile.mk (or Makefile, if you're using nmake) and change 
+the values of INST_DRV and INST_TOP.   You can also enable various
+build flags.  These are explained in the makefiles.
 
-You will have to make sure CCTYPE is set correctly, and CCHOME points
-to wherever you installed your compiler.
+You will have to make sure that CCTYPE is set correctly and that 
+CCHOME points to wherever you installed your compiler.
 
 The default value for CCHOME in the makefiles for Visual C++
 may not be correct for some versions.  Make sure the default exists
@@ -161,7 +161,7 @@
 bundled with the distribution due to US Government restrictions
 on the export of cryptographic software.  Nevertheless, this routine
 is part of the "libdes" library (written by Eric Young) which is widely
-available worldwide, usually along with SSLeay (for example:
+available worldwide, usually along with SSLeay (for example, 
 "ftp://fractal.mta.ca/pub/crypto/SSLeay/DES/").  Set CRYPT_SRC to the
 name of the file that implements des_fcrypt().  Alternatively, if
 you have built a library that contains des_fcrypt(), you can set
@@ -208,7 +208,7 @@
 arising from the inability to find the Borland Runtime DLLs on the system
 default path.  You will need to copy the DLLs reported by the messages
 from where Borland chose to install it, into the Windows system directory
-(usually somewhere like C:\WINNT\SYSTEM32), and rerun the test.
+(usually somewhere like C:\WINNT\SYSTEM32) and rerun the test.
 
 Please report any other failures as described under L<BUGS AND CAVEATS>.
 
@@ -220,7 +220,7 @@
 C<$INST_TOP\$VERSION\lib\pod> and HTML versions of the same under
 C<$INST_TOP\$VERSION\lib\pod\html>.  To use the Perl you just installed,
 you will need to add two components to your PATH environment variable,
-C<$INST_TOP\$VERSION\bin>, and C<$INST_TOP\$VERSION\bin\$ARCHNAME>.
+C<$INST_TOP\$VERSION\bin> and C<$INST_TOP\$VERSION\bin\$ARCHNAME>.
 For example:
 
     set PATH c:\perl\5.6.0\bin;c:\perl\5.6.0\bin\MSWin32-x86;%PATH%
@@ -288,11 +288,12 @@
 expansions of command-line arguments (so wildcards need not be
 quoted).  It also provides only rudimentary quoting.  The only
 (useful) quote character is the double quote (").  It can be used to
-protect spaces in arguments and other special characters.  The
-Windows NT documentation has almost no description of how the
+protect spaces in arguments and other special characters.
+
+The Windows NT documentation has almost no description of how the
 quoting rules are implemented, but here are some general observations
 based on experiments:  The shell breaks arguments at spaces and
-passes them to programs in argc/argv.  Doublequotes can be used
+passes them to programs in argc/argv.  Double quotes can be used
 to prevent arguments with spaces in them from being split up.
 You can put a double quote in an argument by escaping it with
 a backslash and enclosing the whole argument within double quotes.
@@ -300,12 +301,12 @@
 argument will be stripped by the shell.
 
 The file redirection characters "<", ">", and "|" cannot be quoted
-by double quotes (there are probably more such).  Single quotes
-will protect those three file redirection characters, but the
-single quotes don't get stripped by the shell (just to make this
-type of quoting completely useless).  The caret "^" has also
-been observed to behave as a quoting character (and doesn't get
-stripped by the shell also).
+by double quotes (there are probably more).  Single quotes will
+protect those three file redirection characters, but the single 
+quotes don't get stripped by the shell (just to make this type of
+quoting completely useless).  The caret "^" has also been observed
+to behave as a quoting character (and also doesn't get stripped by
+the shell).
 
 Here are some examples of usage of the "cmd" shell:
 
@@ -366,12 +367,12 @@
 
 where $MAKE is whatever 'make' program you have configured perl to
 use.  Use "perl -V:make" to find out what this is.  Some extensions
-may not provide a testsuite (so "$MAKE test" may not do anything, or
+may not provide a testsuite (so "$MAKE test" may not do anything or
 fail), but most serious ones do.
 
 It is important that you use a supported 'make' program, and
 ensure Config.pm knows about it.  If you don't have nmake, you can
-either get dmake from the location mentioned earlier, or get an
+either get dmake from the location mentioned earlier or get an
 old version of nmake reportedly available from:
 
     ftp://ftp.microsoft.com/Softlib/MSLFILES/nmake15.exe
@@ -419,11 +420,11 @@
 alternate shell that *does* expand wildcards.
 
 Instead, the following solution works rather well. The nice things
-about it: 1) you can start using it right away 2) it is more powerful,
-because it will do the right thing with a pattern like */*/*.c
-3) you can decide whether you do/don't want to use it 4) you can
-extend the method to add any customizations (or even entirely
-different kinds of wildcard expansion).
+about it are 1) you can start using it right away; 2) it is more 
+powerful, because it will do the right thing with a pattern like
+*/*/*.c; 3) you can decide whether you do/don't want to use it; and
+4) you can extend the method to add any customizations (or even 
+entirely different kinds of wildcard expansion).
 
        C:\> copy con c:\perl\lib\Wild.pm
        # Wild.pm - emulate shell @ARGV expansion on shells that don't
@@ -465,7 +466,7 @@
 be used under the Activeware port of Perl, which used to be the only
 native port for the Win32 platform.  Since the Activeware port does not
 have adequate support for Perl's extension building tools, these
-extensions typically do not support those tools either, and therefore
+extensions typically do not support those tools either and, therefore,
 cannot be built using the generic steps shown in the previous section.
 
 To ensure smooth transitioning of existing code that uses the
@@ -521,7 +522,7 @@
 refer to all the command line arguments, so you may need to make
 sure that construct works in batch files.  As of this writing,
 4DOS/NT users will need a "ParameterChar = *" statement in their
-4NT.INI file, or will need to execute "setdos /p*" in the 4DOS/NT
+4NT.INI file or will need to execute "setdos /p*" in the 4DOS/NT
 startup file to enable this to work.
 
 =item 3
@@ -575,7 +576,7 @@
 L<perlfunc>, and a few are not implemented at all.  To avoid
 surprises, particularly if you have had prior exposure to Perl
 in other operating environments or if you intend to write code
-that will be portable to other environments, see L<perlport>
+that will be portable to other environments.  See L<perlport>
 for a reasonably definitive list of these differences.
 
 Not all extensions available from CPAN may build or work properly
```
</details>

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

On Mon\, 3 Apr 2000\, A\. C\. Yardley wrote&#8203;:

> @&#8203;@&#8203; \-48\,9 \+48\,9 @&#8203;@&#8203;
>  for it is still experimental\.  \(Older versions of GCC are known
>  not to work\.\)
>  
> \-This port currently supports MakeMaker \(the set of modules that
> \-is used to build extensions to perl\)\.  Therefore\, you should be
> \-able to build and install most extensions found in the CPAN sites\.
> \+This port currently supports MakeMaker \(the set of modules is used
> \+to build extensions to perl\)\.  Therefore\, you should be able to
> \+build and install most extensions found in the CPAN sites\.
>  See L\<Usage Hints> below for general hints about this\.
>  
>  =head2 Setting Up

I'd suggest not applying this bit of the patch \-\- the original seems
correct here\, while the suggested replacement seems wrong\.

Cheers\,
Philip

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

I notice that A\.C Yardley \(yardley@&#8203;tanet\.net\) sent this patch which wasn't
added\.  I'm reforwarding it\.  There was a comment about one section of the
patch \(changing from "the set of modules that is used\.\.\." to "the set of
modules is used\.\.\."\) that I'm not including\.

\-spp

Forwarded&#8203;:

Please note\, most of these are very minor\.  Feel free to do with
 then as you please\.

 Also\, I have two questions regarding README\.win32&#8203;:  \(1\) Do we want
 to change the references to "manpage" to "perldoc"? and \(2\) Do we
 want to include\, where relevant\, references to Windows 2000\, along
 side of the references to Windows NT?

 If so\, please let me know\, and I'll make the corrections\.
 Otherwise\, I'll just leave them as they are\.

 /acy

 \-\-\- README\.win32\.old    Mon Apr 03 07&#8203;:14&#8203;:51 2000
 \+\+\+ README\.win32        Mon Apr 03 11&#8203;:14&#8203;:49 2000
 @&#8203;@&#8203; \-14\,7 \+14\,7 @&#8203;@&#8203;
&nbsp;  =head1 DESCRIPTION
&nbsp;  
&nbsp;  Before you start\, you should glance through the README file
 \-found in the top\-level directory where the Perl distribution
 \+found in the top\-level directory to which the Perl distribution
&nbsp;  was extracted\.  Make sure you read and understand the terms under
&nbsp;  which this software is being distributed\.
&nbsp;  
 @&#8203;@&#8203; \-28\,10 \+28\,10 @&#8203;@&#8203;

&nbsp;  You may also want to look at two other options for building
&nbsp;  a perl that will work on Windows NT&#8203;:  the README\.cygwin and
 \-README\.os2 files\, which each give a different set of rules to build
 \-a Perl that will work on Win32 platforms\.  Those two methods will
 \-probably enable you to build a more Unix\-compatible perl\, but you
 \-will also need to download and use various other build\-time and
 \+README\.os2 files\, each of which give a different set of rules to
 \+build a Perl that will work on Win32 platforms\.  Those two methods
 \+will probably enable you to build a more Unix\-compatible perl\, but
 \+you will also need to download and use various other build\-time and
&nbsp;  run\-time support software described in those files\.
&nbsp;  
&nbsp;  This set of instructions is meant to describe a so\-called "native"
@&#8203;@&#8203; \-70\,9 \+70\,9 @&#8203;@&#8203;
&nbsp;  
&nbsp;      http&#8203;://www\.cpan\.org/authors/id/GSAR/dmake\-4\.1pl1\-win32\.zip
&nbsp;  
 \-\(This is a fixed version of original dmake sources obtained from
 \+\(This is a fixed version of the original dmake sources obtained from
&nbsp;  http&#8203;://www\.wticorp\.com/dmake/\.  As of version 4\.1PL1\, the original
 \-sources did not build as shipped\, and had various other problems\.
 \+sources did not build as shipped and had various other problems\.
&nbsp;  A patch is included in the above fixed version\.\)
&nbsp;  
&nbsp;  Fetch and install dmake somewhere on your path \(follow the instructions
@&#8203;@&#8203; \-97\,20 \+97\,20 @&#8203;@&#8203;
&nbsp;  =item Borland C\+\+
&nbsp;  
&nbsp;  If you are using the Borland compiler\, you will need dmake\.
 \-\(The make that Borland supplies is seriously crippled\, and will not
 \+\(The make that Borland supplies is seriously crippled and will not
&nbsp;  work for MakeMaker builds\.\)
&nbsp;  
 \-See L/"Make"> above\.
 \+See L\</"Make"> above\.
&nbsp;  
&nbsp;  =item Microsoft Visual C\+\+
&nbsp;  
&nbsp;  The nmake that comes with Visual C\+\+ will suffice for building\.
 \-You will need to run the VCVARS32\.BAT file usually found somewhere
 \+You will need to run the VCVARS32\.BAT file\, usually found somewhere
&nbsp;  like C&#8203;:\\MSDEV4\.2\\BIN\.  This will set your build environment\.
&nbsp;  
 \-You can also use dmake to build using Visual C\+\+\, provided&#8203;:
 \+You can also use dmake to build using Visual C\+\+; provided\, however\,
&nbsp;  you set OSRELEASE to "microsft" \(or whatever the directory name
 \-under which the Visual C dmake configuration lives\) in your environment\,
 \+under which the Visual C dmake configuration lives\) in your environment
&nbsp;  and edit win32/config\.vc to change "make=nmake" into "make=dmake"\.  The
&nbsp;  latter step is only essential if you want to use dmake as your default
&nbsp;  make for building extensions using MakeMaker\.
@&#8203;@&#8203; \-125\,7 \+125\,7 @&#8203;@&#8203;
&nbsp;  
&nbsp;  Make sure you install the binaries that work with MSVCRT\.DLL as indicated
&nbsp;  in the README for the GCC bundle\.  You may need to set up a few
environment
 \-variables \(usually run from a batch file\)\.
 \+variables \(usually ran from a batch file\)\.
&nbsp;  
&nbsp;  You also need dmake\.  See L\</"Make"> above on how to get it\.
&nbsp;  
 @&#8203;@&#8203; \-145\,12 \+145\,12 @&#8203;@&#8203;
&nbsp;  
&nbsp;  =item \*
&nbsp;  
 \-Edit the makefile\.mk \(or Makefile\, if using nmake\) and change the values
 \-of INST\_DRV and INST\_TOP\.   You can also enable various build
 \-flags\.  These are explained in the makefiles\.
 \+Edit the makefile\.mk \(or Makefile\, if you're using nmake\) and change 
 \+the values of INST\_DRV and INST\_TOP\.   You can also enable various
 \+build flags\.  These are explained in the makefiles\.
&nbsp;  
 \-You will have to make sure CCTYPE is set correctly\, and CCHOME points
 \-to wherever you installed your compiler\.
 \+You will have to make sure that CCTYPE is set correctly and that 
 \+CCHOME points to wherever you installed your compiler\.
&nbsp;  
&nbsp;  The default value for CCHOME in the makefiles for Visual C\+\+
&nbsp;  may not be correct for some versions\.  Make sure the default exists
 @&#8203;@&#8203; \-161\,7 \+161\,7 @&#8203;@&#8203;
&nbsp;  bundled with the distribution due to US Government restrictions
&nbsp;  on the export of cryptographic software\.  Nevertheless\, this routine
&nbsp;  is part of the "libdes" library \(written by Eric Young\) which is widely
 \-available worldwide\, usually along with SSLeay \(for example&#8203;:
 \+available worldwide\, usually along with SSLeay \(for example\, 
&nbsp;  "ftp&#8203;://fractal\.mta\.ca/pub/crypto/SSLeay/DES/"\)\.  Set CRYPT\_SRC to the
&nbsp;  name of the file that implements des\_fcrypt\(\)\.  Alternatively\, if
&nbsp;  you have built a library that contains des\_fcrypt\(\)\, you can set
 @&#8203;@&#8203; \-208\,7 \+208\,7 @&#8203;@&#8203;
&nbsp;  arising from the inability to find the Borland Runtime DLLs on the system
&nbsp;  default path\.  You will need to copy the DLLs reported by the messages
&nbsp;  from where Borland chose to install it\, into the Windows system directory
 \-\(usually somewhere like C&#8203;:\\WINNT\\SYSTEM32\)\, and rerun the test\.
 \+\(usually somewhere like C&#8203;:\\WINNT\\SYSTEM32\) and rerun the test\.
 
&nbsp;  Please report any other failures as described under L\<BUGS AND CAVEATS>\.
&nbsp;  
 @&#8203;@&#8203; \-220\,7 \+220\,7 @&#8203;@&#8203;
&nbsp;  C\<$INST\_TOP\\$VERSION\\lib\\pod> and HTML versions of the same under
&nbsp;  C\<$INST\_TOP\\$VERSION\\lib\\pod\\html>\.  To use the Perl you just installed\,
&nbsp;  you will need to add two components to your PATH environment variable\,
 \-C\<$INST\_TOP\\$VERSION\\bin>\, and C\<$INST\_TOP\\$VERSION\\bin\\$ARCHNAME>\.
 \+C\<$INST\_TOP\\$VERSION\\bin> and C\<$INST\_TOP\\$VERSION\\bin\\$ARCHNAME>\.
&nbsp;  For example&#8203;:
&nbsp;  
&nbsp;      set PATH c&#8203;:\\perl\\5\.6\.0\\bin;c&#8203;:\\perl\\5\.6\.0\\bin\\MSWin32\-x86;%PATH%
 @&#8203;@&#8203; \-288\,11 \+288\,12 @&#8203;@&#8203;
&nbsp;  expansions of command\-line arguments \(so wildcards need not be
&nbsp;  quoted\)\.  It also provides only rudimentary quoting\.  The only
&nbsp;  \(useful\) quote character is the double quote \("\)\.  It can be used to
 \-protect spaces in arguments and other special characters\.  The
 \-Windows NT documentation has almost no description of how the
 \+protect spaces in arguments and other special characters\.
 \+
 \+The Windows NT documentation has almost no description of how the
&nbsp;  quoting rules are implemented\, but here are some general observations
&nbsp;  based on experiments&#8203;:  The shell breaks arguments at spaces and
 \-passes them to programs in argc/argv\.  Doublequotes can be used
 \+passes them to programs in argc/argv\.  Double quotes can be used
&nbsp;  to prevent arguments with spaces in them from being split up\.
&nbsp;  You can put a double quote in an argument by escaping it with
&nbsp;  a backslash and enclosing the whole argument within double quotes\.
 @&#8203;@&#8203; \-300\,12 \+301\,12 @&#8203;@&#8203;
&nbsp;  argument will be stripped by the shell\.
&nbsp;  
&nbsp;  The file redirection characters "\<"\, ">"\, and "|" cannot be quoted
 \-by double quotes \(there are probably more such\)\.  Single quotes
 \-will protect those three file redirection characters\, but the
 \-single quotes don't get stripped by the shell \(just to make this
 \-type of quoting completely useless\)\.  The caret "^" has also
 \-been observed to behave as a quoting character \(and doesn't get
 \-stripped by the shell also\)\.
 \+by double quotes \(there are probably more\)\.  Single quotes will
 \+protect those three file redirection characters\, but the single 
 \+quotes don't get stripped by the shell \(just to make this type of
 \+quoting completely useless\)\.  The caret "^" has also been observed
 \+to behave as a quoting character \(and also doesn't get stripped by
\+the shell\)\.
&nbsp;  
&nbsp;  Here are some examples of usage of the "cmd" shell&#8203;:
&nbsp;  
 @&#8203;@&#8203; \-366\,12 \+367\,12 @&#8203;@&#8203;
&nbsp;  
&nbsp;  where $MAKE is whatever 'make' program you have configured perl to
&nbsp;  use\.  Use "perl \-V&#8203;:make" to find out what this is\.  Some extensions
 \-may not provide a testsuite \(so "$MAKE test" may not do anything\, or
 \+may not provide a testsuite \(so "$MAKE test" may not do anything or
&nbsp;  fail\)\, but most serious ones do\.
&nbsp;  
&nbsp;  It is important that you use a supported 'make' program\, and
&nbsp;  ensure Config\.pm knows about it\.  If you don't have nmake\, you can
 \-either get dmake from the location mentioned earlier\, or get an
 \+either get dmake from the location mentioned earlier or get an
&nbsp;  old version of nmake reportedly available from&#8203;:
&nbsp;  
&nbsp;      ftp&#8203;://ftp\.microsoft\.com/Softlib/MSLFILES/nmake15\.exe
 @&#8203;@&#8203; \-419\,11 \+420\,11 @&#8203;@&#8203;
&nbsp;  alternate shell that \*does\* expand wildcards\.
&nbsp;  
&nbsp;  Instead\, the following solution works rather well\. The nice things
 \-about it&#8203;: 1\) you can start using it right away 2\) it is more powerful\,
 \-because it will do the right thing with a pattern like \*/\*/\*\.c
 \-3\) you can decide whether you do/don't want to use it 4\) you can
 \-extend the method to add any customizations \(or even entirely
 \-different kinds of wildcard expansion\)\.
 \+about it are 1\) you can start using it right away; 2\) it is more 
 \+powerful\, because it will do the right thing with a pattern like
 \+\*/\*/\*\.c; 3\) you can decide whether you do/don't want to use it; and
 \+4\) you can extend the method to add any customizations \(or even 
 \+entirely different kinds of wildcard expansion\)\.
&nbsp;  
&nbsp;         C&#8203;:\\> copy con c&#8203;:\\perl\\lib\\Wild\.pm
&nbsp;         \# Wild\.pm \- emulate shell @&#8203;ARGV expansion on shells that don't
 @&#8203;@&#8203; \-465\,7 \+466\,7 @&#8203;@&#8203;
&nbsp;  be used under the Activeware port of Perl\, which used to be the only
&nbsp;  native port for the Win32 platform\.  Since the Activeware port does not
&nbsp;  have adequate support for Perl's extension building tools\, these
 \-extensions typically do not support those tools either\, and therefore
 \+extensions typically do not support those tools either and\, therefore\,
&nbsp;  cannot be built using the generic steps shown in the previous section\.
 
&nbsp;  To ensure smooth transitioning of existing code that uses the
 @&#8203;@&#8203; \-521\,7 \+522\,7 @&#8203;@&#8203;
&nbsp;  refer to all the command line arguments\, so you may need to make
&nbsp;  sure that construct works in batch files\.  As of this writing\,
&nbsp;  4DOS/NT users will need a "ParameterChar = \*" statement in their
 \-4NT\.INI file\, or will need to execute "setdos /p\*" in the 4DOS/NT
 \+4NT\.INI file or will need to execute "setdos /p\*" in the 4DOS/NT
&nbsp;  startup file to enable this to work\.
&nbsp;  
&nbsp;  =item 3
 @&#8203;@&#8203; \-575\,7 \+576\,7 @&#8203;@&#8203;
&nbsp;  L\<perlfunc>\, and a few are not implemented at all\.  To avoid
&nbsp;  surprises\, particularly if you have had prior exposure to Perl
&nbsp;  in other operating environments or if you intend to write code
 \-that will be portable to other environments\, see L\<perlport>
 \+that will be portable to other environments\.  See L\<perlport>
&nbsp;  for a reasonably definitive list of these differences\.
&nbsp;  
&nbsp;  Not all extensions available from CPAN may build or work properly

**Comment by p5pRT** [comment]
### From @jhi

On Fri\, Nov 17\, 2000 at 11&#8203;:30&#8203;:01AM \-0500\, Stephen P\. Potter wrote&#8203;:
> I notice that A\.C Yardley \(yardley@&#8203;tanet\.net\) sent this patch which wasn't
> added\.  I'm reforwarding it\.  There was a comment about one section of the
> patch \(changing from "the set of modules that is used\.\.\." to "the set of
> modules is used\.\.\."\) that I'm not including\.

Applied\, thanks\.
